homeassistant/components/climate/opentherm_gw.py

```diff
@@ -0,0 +1,201 @@
+"""OpenTherm Gateway Climate component for Home Assistant."""
+
```

Comment:
Remove blank line.

Reply:
addressed in c5a9266871b32a981ad19ab38a0bad650ab644e4